site/src/pages/components/selectmenu.mdx

```diff
@@ -52,18 +52,18 @@ Selectmenu provides a variety of tools to help with styling, including pseudo-se
 
 ### Styling parts of the control
 
-One way to style the control to match your requirements is to use the CSS `::part()` pseudo-element to select the different parts within the control's anatomy that you wish to style.
+One way to style the control to match your requirements is to use the CSS `::behavior()` pseudo-element, newly introduced for the selectmenu element, to select the different parts within the control's anatomy that you wish to style.
```

Comment:
We should link out to the definition of behavior. Do we have one in CSSWG yet? If not we should probably expound on this.

Reply:
I don't know of a CSSWG issue proposing `::behavior` yet.
If we remove the "newly introduced for the selectmenu element" part is that good enough to merge this now?
I'm hesitant to open a CSSWG issue proposing this just yet since domenic's general feedback could change how this works. I haven't even opened an HTML issue for selectmenu yet.

Reply:
This is not in CSSWG yet.

> I'm hesitant to open a CSSWG issue proposing this just yet since domenic's general feedback could change how this works. I haven't even opened an HTML issue for selectmenu yet.

I feel the same. Should we create a draft spec of `::behavior` somewhere at open-ui?